TYPO3.Flow/Classes/TYPO3/Flow/Http/Client/CurlEngine.php

```diff
@@ -148,9 +148,10 @@ public function sendRequest(Http\Request $request)
         curl_close($curlHandle);
 
         $response = Http\Response::createFromRaw($curlResult);
-        if ($response->getStatusCode() === 100) {
+        while (substr($response->getContent(), 0, 5) === 'HTTP/' || $response->getStatusCode() === 100) {
```

Comment:
Actually not sure if the `100` code check is still needed with this, but I kept it in place just to be sure.